flamingpy/codes/surface_code.py

```diff
@@ -285,6 +300,8 @@ def __init__(
 
         if boundaries == "open":
             self.bound_str = "open_primal" if ec in ("primal", "both") else "open_dual"
```

Comment:
```suggestion
            self.bound_str = "open_dual" if ec in ("primal", "both") else "open_primal"
```
(Just to align with our option relabelling)